app/data-sources/authenticate/AuthenticateDatabase.js

```diff
@@ -1,49 +1,48 @@
-import Sequelize, { DataTypes } from 'sequelize'
+import { Sequelize, DataTypes } from 'sequelize'
 import { logger } from '../../utils/logger.js'
 
-const databaseName = process.env.AUTHENTICATE_DB_SCHEMA
-const serverHost = process.env.AUTHENTICATE_DB_HOST
-
-const readUsername = process.env.AUTHENTICATE_DB_USERNAME
-const readPassword = process.env.AUTHENTICATE_DB_PASSWORD
-const sequelizeRead = new Sequelize(databaseName, readUsername, readPassword, {
-  host: serverHost,
+const dbOptions = {
+  database: process.env.AUTHENTICATE_DB_SCHEMA,
+  host: process.env.AUTHENTICATE_DB_HOST,
   dialect: 'mssql',
   dialectOptions: { options: { encrypt: false } },
```

Comment:
`encrypt: false` really!?

Reply:
Not sure… @defra-dom do you know? I just copied over the settings from your original implementation. When working locally with docker, doesn't seem to make much difference if `true` or `false`.

Reply:
cant remember, will be somthing to do with connecting to auth db in live